test/e2e/resources/tls/server.csr.conf

```diff
@@ -53,7 +53,149 @@ DNS.38 = *.etcd-local-peer.shoot
 DNS.39 = *.etcd-local-peer.shoot.svc
 DNS.40 = *.etcd-local-peer.shoot.svc.cluster.local
 
+# Multi-node-etcd entries
+DNS.41 = multi-node-etcd-0
```

Comment:
All this is not really required.
In future we will generate fresh PKI material for every etcd cluster and then none of this is required.

Reply:
Yeah, I agree.